packages/web/src/components/spa/MembersSection/Carousel.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import Slider from 'react-slick';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import './carousel.css';
+import { FaLinkedin } from 'react-icons/fa';
+
+export default class SimpleSlider extends Component {
+    constructor(props) {
```

Comment:
Avoid using jquery. Look at how you have implemented it in the HackAUBG page and do it in the same way here